scripts/requirements-actions.txt

```diff
@@ -1,5 +1,5 @@
 # This file was autogenerated by uv via the following command:
-#    uv pip compile --universal --python-version 3.10 --generate-hashes requirements-actions.in --output-file requirements-actions.txt
```

Comment:
The file should be refreshed by running the exact same command (i.e targeting python 3.10)

Reply:
Ok, will fix it. If we wont provide the esptool as a package we will be force to use it like `PYTHONPATH=/path/to/hal/espressif/tools/ python -m esptool` instead of `esptool`. If you see the other options please comment.